lib/meta-boxes.php

```diff
@@ -84,5 +84,96 @@ function igv_cmb_metaboxes() {
     'type'       => 'text_url',
   ) );
 
+  // HOME PAGE METABOXES
+  $home_page = get_page_by_path('home');
+
+  if (!empty($home_page) ) {
+    $cmb_home = new_cmb2_box( array(
+      'id'            => $prefix . 'home_metabox',
+      'title'         => esc_html__( 'Options', 'cmb2' ),
+      'object_types'  => array( 'page', ), // Post type
+      'show_on'      => array( 'key' => 'id', 'value' => array($home_page->ID) ),
+    ) );
+
+    $cmb_home->add_field( array(
+      'name'       => esc_html__( 'Splash Text', 'cmb2' ),
+      'id'         => $prefix . 'splash_text',
+      'type'       => 'textarea_small',
+    ) );
+
+    $cmb_home->add_field( array(
+      'name'       => esc_html__( 'Splash Image', 'cmb2' ),
+      'id'         => $prefix . 'splash_image',
+      'type'       => 'file',
+    ) );
+
+    // Get quote posts, to be used as slect field values
+    $press_quote_posts = get_posts_as_options('press_quote');
+
+    // Generate this field 3 times
+    for($i = 1; $i <= 3; $i++) {
+      $cmb_home->add_field( array(
+        'name'       => esc_html__( 'Press Quote #' . $i, 'cmb2' ),
+        'id'         => $prefix . 'home_press_quote_' . $i,
+        'type'       => 'select',
+        'show_option_none' => true,
+        'options' => $press_quote_posts,
+      ) );
+    }
+  }
+
+  // WHAT WE DO PAGE METABOXES
+  $what_we_do_page = get_page_by_path('what-we-do');
+
+  if (!empty($what_we_do_page) ) {
+    $cmb_what_we_do = new_cmb2_box( array(
+      'id'            => $prefix . 'what_we_do_metabox',
+      'title'         => esc_html__( 'Homepage options', 'cmb2' ),
+      'object_types'  => array( 'page', ), // Post type
+      'show_on'      => array( 'key' => 'id', 'value' => array($what_we_do_page->ID) ),
+    ) );
+  }
+
+  $cmb_what_we_do->add_field( array(
```

Comment:
Which will always happen if the pages are not created. Same for the rest. So the pages either need to be created in the theme with a hook and/or this conditionals fixed